lib/env.js

```diff
@@ -48,3 +48,12 @@ export function getEnvironment(pathPrefix) {
 
   return { raw, stringified };
 }
+
+export function getBasePath(domain) {
```

Comment:
This is the key function. Everything in the chain goes back to using this logic to determine what basePath should be used

Reply:
looking at line 9,
```js
export const isProductionEnv = nodeEnv === 'production';
```
`isProductionEnv` does not determine if it's local dev because it's always production

Reply:
it does determine dev. `nodeEnv === 'production'` is a comparison not an assignment